test/unit/resources/cpan_test.rb

```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::Cpan' do
+  it 'verify cpan package detail parsing' do
```

Comment:
We should add a negative test as well, ensuring it returns the appropriate messages when a package is not installed. Let me know if you need guidance on doing that.